### ISSUE TITLE
Idris requres (at least) node 10.4.0 due BigInt

### ISSUE THREAD

**Issue by phadej** [idris-lang/Idris2#861]
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt

INSTALL.md doesn't mention NodeJS at all (e.g. if it is possible to disable without uninstalling old node which is my machine).

System: Ubuntu 18.04 (which bundles nodejs-8.10.0).

**Comment by gallais** [comment]
Ugh, that's inconvenient.

You should still be able to run `make install` after the `make bootstrap`
even if the test suite run failed because of node-related issues.
You should then have a working installation of Idris2.

**Comment by phadej** [comment]
Yeah, I should just update to Ubuntu 20.04 (current LTS), but there is other stuff which would break do that :)

**Comment by phadej** [comment]
> The javascript codegen uses the new BigInt, hence nodejs 10.4 or higher is required. 

But it's in README.md, not INSTALL.md
